### ISSUE TITLE
memberlist log entries nested inside another layer of logging

### ISSUE THREAD

**Issue by aaronlehmann** [docker/docker#28969]
These messages are weird and hard to read:

```
Nov 22 16:34:16 hostname dockerd: time="2016-11-22T16:34:16.802103258-08:00" level=info msg="2016/11/22 16:34:16 [INFO] memberlist: Marking xyz-1d1ec2dfa053 as failed, suspect timeout reached\n"
```

The "msg" filed includes a complete log line, with a second timestamp and log level. There is a newline at the end which is escaped.

I would expect us to parse out the useful information from this log entry and log that, intead of nesting two complete log lines. At a minimum, we should strip the newline.

Possibly caused by code in libnetwork.

**Comment by aaronlehmann** [comment]
Opened https://github.com/docker/libnetwork/pull/1582

**Comment by aboch** [comment]
Fixed by #29319
